src/modules/launcher/Plugins/Microsoft.Plugin.Shell/ShellPluginSettings.cs

```diff
@@ -31,13 +34,34 @@ public void AddCmdHistory(string cmdName)
                 Count.Add(cmdName, 1);
             }
         }
+
+        public static string GetDescription(Enum value)
+        {
+            FieldInfo fi = value.GetType().GetField(value.ToString());
+
+            DescriptionAttribute[] attributes =
+                (DescriptionAttribute[])fi.GetCustomAttributes(typeof(DescriptionAttribute), false);
+
+            if (attributes != null && attributes.Length > 0)
+            {
+                return attributes[0].Description;
+            }
+            else
+            {
+                return value.ToString();
+            }
+        }
     }
 
     public enum ExecutionShell
     {
+        [Description("Run command in Command Prompt (cmd.exe)")]
```

Comment:
These are user-facing strings and have to be localized

Reply:
Done.